### PR TITLE
Make data fetches on path parameter

### PR DESCRIPTION
**Pull request by cmonez**
Also changed div tags where services are react.dom rendered to

### CODE REVIEW

**Comment by cmonez** [comment]
@samasastudio @andyoe0102 @krisdo

File: client/Index.jsx [hunk 1]
```diff
@@ -14,12 +14,14 @@ class Index extends React.Component {
       amenities: [],
       hostAndRooms: {},
       thingsToDo: [],
+      listingId: window.location.href.split('/')[3] || 0,
     };
```
Comment (samasastudio) [inline client/Index.jsx:18]: try refactoring this to use window.location.pathname

File: server/app.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const db = require('../database/connection');
 const Document = require('../database/Description');
 const app = express();
 
-app.use(express.static('public'));
+app.use('/:listingId', express.static('public'));
 
 app.get('/api/description/:listingId', (req, res) => {
```
Comment (samasastudio) [inline server/app.js:8]: doing this myself now

File: client/Index.jsx (hunk 1 above)
Comment (cmonez) [reply client/Index.jsx:18]: I tried but I didnt wanna deal with the trailing "/" , will do.

File: server/app.js (hunk 2 above)
Comment (cmonez) [reply server/app.js:8]: Good to hear!

**Pull request merged**